example/example.dart

```diff
@@ -3,13 +3,15 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:code_builder/code_builder.dart';
+import 'package:code_builder/src/specs/enum.dart';
```

Comment:
The examples shouldn't need to import from `src`. Is there something else that needs to be exported?

Reply:
okay this import was useless.